azurerm/resource_arm_iothub.go

```diff
@@ -308,6 +315,12 @@ func resourceArmIotHubCreateUpdate(d *schema.ResourceData, meta interface{}) err
 			Routing: &devices.RoutingProperties{
 				Endpoints: endpoints,
 				Routes:    routes,
+				FallbackRoute: &devices.FallbackRouteProperties{
+					Name:          utils.String("$fallback"),
```

Comment:
Shouldn't we allow the user to set these and configure the default values to match the portal?

Reply:
Modified it so the defaults match the portal and a user can set their own values.
For the fallback route there can only be 1 endpoint so added that as a string with the default value, instead of a list.